library/src/main/java/com/bumptech/glide/manager/SupportRequestManagerFragment.java

```diff
@@ -20,6 +21,9 @@
  * @see com.bumptech.glide.RequestManager
  */
 public class SupportRequestManagerFragment extends Fragment {
+
+    private static final String TAG = SupportRequestManagerFragment.class.getSimpleName();
```

Comment:
oh, so that's why he does not simply use getSimpleName. Good to know, I will update the PR.